src/analyses/ai.h

```diff
@@ -398,7 +395,8 @@ template<typename domainT>
 class concurrency_aware_ait:public ait<domainT>
 {
 public:
-  typedef typename ait<domainT>::statet statet;
+  using statet = typename ait<domainT>::statet;
```

Comment:
@kroening The coding standard currently says "Prefer use of typedef instead of using" but I see you prefer `using` here. Should the coding standard be updated?